packages/clay-dropdown/src/ClayDropdownBase.js

```diff
@@ -14,11 +14,8 @@ import templates from './ClayDropdownBase.soy.js';
 
 const KEY_CODE_ESC = 27;
 
-const KEY_DOWN = 'ArrowDown';
```

Comment:
Are we removing the interactions by the arrows due to UX change in MultiSelect?

Reply:
Yes